### PR TITLE
All tests green!

### PR DESCRIPTION
**Pull request by polinamarchenko**

### CODE REVIEW

File: testing_exercise/testing.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  for (var i = 0; i < num; i++) {
+    newArr = newArr.concat(arr);
+  }
+
+  return newArr;
+}
+
+function acceptNumbersOnly() {
+  for (var arg = 0; arg < arguments.length; arg++) {
+    if (isNaN(arguments[arg]) || typeof arguments[arg] !== "number") {
```
Comment (elie) [inline testing_exercise/testing.js]: check out ES2015's `Number.isFinite` for an easiest way to write this line!

File: testing_exercise/testing.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+function replaceWith(str, char1, char2) {
+  var newStr = "";
+
```
Comment (elie) [inline testing_exercise/testing.js]: Try to start being consistent with your spacing after defining variables. No new lines after each one.

File: testing_exercise/testing.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+function mergeArrays(arr1, arr2) {
+  var newArr = arr1.concat(arr2);
+  return newArr.sort(function(a,b) {
+    return a - b;
+  });
+}
+
+function mergeObjects(obj1, obj2) {
+  for (var key1 in obj1) {
+
```
Comment (elie) [inline testing_exercise/testing.js]: Same idea here, no additional new line

**Comment by polinamarchenko** [comment]
@tigarcia hey Tim, I've just committed the canvas game.

**Pull request closed without merging**